components/nds-full-bleed/scss/full-bleed.scss

```diff
@@ -0,0 +1,32 @@
+@import '~@nice-digital/nds-core/scss/core';
+
+// Multipliers of $nds-spacing-large
+$paddingSizes: (
+  'small': 1,
+  'medium': 3,
+  'large': 6
+);
+
+.full-bleed {
+  background-color: $nds-colour-nice-white;
+  background-position: center center;
+  background-size: cover;
+  left: 50%;
+  margin-left: -50vw;
```

Comment:
Does this work without the margin right? Just curious as hero has one:

https://github.com/nice-digital/nice-design-system/blob/36a76d866bee896e6779ddc3a7cc89ec8e4656bf/components/nds-hero/scss/hero.scss#L12-L15

Reply:
it does!

<img width="2524" alt="Screenshot 2021-03-09 at 09 51 51" src="https://user-images.githubusercontent.com/6341474/110452890-8399c880-80bd-11eb-9565-952a31162c7e.png">